plugins/code-climate/src/api/production-api.ts

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  CodeClimateData,
+  CodeClimateRepoData,
+  CodeClimateMaintainabilityData,
+  CodeClimateTestCoverageData,
+  CodeClimateIssuesData,
+} from './code-climate-data';
+import { CodeClimateApi } from './code-climate-api';
+import { DiscoveryApi, IdentityApi } from '@backstage/core-plugin-api';
+import { Duration } from 'luxon';
+import humanizeDuration from 'humanize-duration';
+
+const basicIssuesQuery = `${encodeURI('page[size]')}=1&${encodeURI(
```

Comment:
Could you double check that these shouldn't be `encodeURIComponent`? in my experience `encodeURI` is almost never what you actually want.

Reply:
Another way of creating queries like this in a pretty convenient way is using `new URLSearchParams` etc, or the `qs` library which we make a fair bit of use of in the repo. They will handle all of the encoding and stuff for you.